문제2.py

```diff
@@ -0,0 +1,7 @@
+T = int(input())
+for i in range(T):
+    a, b, c = map(int,input().split())
+    if a==b:
+        print(f"#{i+1} {c}")
+    else:
+        print(f"#{i+1} {int((a**2+b**2-c**2)**(1/2))}")
```

Comment:
좋은 풀이네요! 👍 
```suggestion
        print(f"#{i+1} {a + b - c}")
```
제곱을 사용하지 않고 답을 출력하는 방법도 공유 드릴게요! 😄